scripts/prepare_dev_workspace.sh

```diff
@@ -19,9 +19,9 @@ set -o verbose
 
 . /opt/ros/${ROS_DISTRO}/setup.sh
 
-# Create a new workspace in 'catkin_ws'.
-mkdir -p catkin_ws/src
-cd catkin_ws/src
+# Create a new workspace in 'dev_ws'.
+mkdir -p dev_ws/src
+cd dev_ws/src
 wstool init
```

Comment:
This is probably a good opportunity to switch from `wstool` -> `vcstool`, since `wstool` is indeed deprecated (and now unmaintained).

Reply:
I would do this this as a separate change because it also needs to be done in the ROS1 code. I made a todo issue: #1549